ofrak_core/Dockerstub

```diff
@@ -27,6 +27,23 @@ RUN wget https://raw.githubusercontent.com/iBotPeaches/Apktool/v2.3.3/scripts/li
   wget https://github.com/patrickfav/uber-apk-signer/releases/download/v1.0.0/uber-apk-signer-1.0.0.jar -O /usr/local/bin/uber-apk-signer.jar && \
   chmod +x /usr/local/bin/apktool*
 
+# Install official 7-zip
```

Comment:
With these changes, do we end up supporting more platforms, fewer, or the same? That is, does `p7zip-full` only support the arm64 and amd64 target architectures?

Reply:
We do end up supporting fewer platforms, as the apt package version of `p7zip-full` supports armhf, ppc64el, and s390x in addition to arm64 and amd64.

Reply:
For the time being, I don't think this should be a blocker – I think we should move ahead with merging this, assuming the conflicts with the `CHANGELOG.md` are resolved.

Reply:
What wasn't clear to me from this PR and the linked issue is that `p7zip` is no longer maintained. I suppose it's worth losing support for some uncommon platforms and a harder dependency install to get the "right" package.